src/js/client/main.js

```diff
@@ -87,8 +87,11 @@ $(document).ready(function () {
     $(window).scroll(() => {
       if (resize) {
         window.requestAnimationFrame(() => {
-          document.querySelector('.landing-welcome-background-box1').style.setProperty('--diamond-scale', `${1 + window.scrollY / 1000}`);
-          resize = true;
+          const diamond = document.querySelector('.landing-welcome-background-box1');
```

Comment:
I love that we're not relying on jQuery for this.

Not a blocker, just interested in your opinion. Do you think it would be better to remove the null checks by making it a bit more functional?

```js
[...document.querySelectorAll('.landing-welcome-background-box1')].forEach(diamond => {
    // etc...
});
```